topics-and-advance-readings/Distributed Social Web Building Blocks.md

```diff
@@ -0,0 +1,25 @@
+Kaliya Young and Ben Werdmuller are working on a paper to outline all of the various protocols innovated in the past 15 years to support the idea of a distributed social web. We have had one brainstroming session so far and plan to meet at least twice more before RWT in October. The paper should serve as a resource to consdier how we actually build socail and interative tools with this new infrastructure. As we all know names to nowhere are not useful. I think it is important to start. 
+
+Distributed Social Web Things
+* XFN (Dead?)
+* MF2 
+* Schema.org
+* RSS & Atom 
+* WebSub (the new name for PubSubHubBub)
```

Comment:
Missing ActivityPub ... /cc @cwebber ... who may may want to add as a collaborator on this Rebooting Web of Trust paper, @Identitywoman